api/envoy/config/core/v3/health_check.proto

```diff
@@ -354,4 +354,15 @@ message HealthCheck {
   // the cluster's :ref:`transport socket <envoy_api_field_config.cluster.v3.Cluster.transport_socket>`
   // will be used for health check socket configuration.
   google.protobuf.Struct transport_socket_match_criteria = 23;
+
+  // The "no traffic healthy interval" is a special health check interval that is used when a cluster has
+  // never had traffic routed to it but is a healthy cluster.
+  // This lower interval allows cluster information to be kept up to
+  // date, without sending a potentially large amount of active health checking traffic for no
+  // reason. Once a cluster has been used for traffic routing, Envoy will shift back to using the
```

Comment:
I think the common case for `no_traffic_interval` is to send *less* frequent health checks, so I think this would be better if we explicitly detailed the use case you're using this for: using a short `no_traffic_interval` to ensure that you detect healthy quickly, but then lowering the interval once the host is known healthy.